### PR TITLE
e2e test to validate persistence of the OVN databases

### PR DESCRIPTION
**Pull request by OmerBenHayun**
this PR continues #1918 .

**- What this PR does and why is it needed**

this PR is adding an e2e test that will check that the database is saved on the node file system (and not on a particular pod file system). The test will delete all the db pods (1 pod when on `non-ha` mode or 3 pods when on `ha` mode) without deleting any database files. After all the db pods will be be ready again we will test that new pod can be created and pass simple connectivity test to the API server.

**- Special notes for reviewers**

as said earlier this PR continues #1918 .

the test validate connectivity before and after deleting all the db pods .
(we are  deleting the pods in order to emulate the pod restart.)

**- How to verify it**

in order to try it yourself you can run from the `test` directory:
```
make control-plane WHAT="Should validate connectivity before and after deleting all the db-pods at once"
```

**- Description for the changelog**

adding e2e test to validate persistence of the OVN databases.

### CODE REVIEW

**Comment by girishmg** [comment]
@OmerBenHayun now that PR 1918 is in this needs rebase and re-merge. @roytman FYI

**Comment by OmerBenHayun** [comment]
@girishmg thanks, I rebased it.

File: test/e2e/e2e.go [hunk 1]
```diff
@@ -2307,4 +2307,30 @@ var _ = ginkgo.Describe("e2e delete databases", func() {
...
+		singlePodConnectivityTest(f, "before-delete-db-pods")
+
+		framework.Logf("deleting all the db pods")
+		for _, dbPod := range dbPodList {
+			dbPodName := dbPod.Name
+			framework.Logf("deleting db pod: %v", dbPodName)
+			//delete the db-pod in order to emulate the pod restart
+			deletePod(f, ovnNs, dbPodName)
+		}
+		time.Sleep(waitingPeriod)
```
[the remark this answers is not in the document]
Comment (OmerBenHayun) [reply test/e2e/e2e.go]: Thanks @girishmg , fixed this.
please notice that there is another place on previous `e2e` that I used ` time.Sleep(waitingPeriod)` too (2215-2217 on `e2e.go`):
```go
//wait (let the new db pod to be ready)
framework.Logf("wait %v",waitingPeriod)
time.Sleep(waitingPeriod)
```
I will fix this issue on another PR.

**Pull request merged**